doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -257,20 +264,21 @@ Some operations imply that only a partial representation of an entity will be pr
 
 * `id` and `type` are not allowed in update operations, as they are inmutable properties.
 
-* In the requests in which entity `type` is allowed, it may be omitted. When omitted, the default string value `none`
-  is used for the type.
```

Comment:
To be discussed in a next PR. 

NTC for this PR with regard this (eventual future modifications would be done in a next PR).